pkg/resources/schema_acceptance_test.go

```diff
@@ -1044,3 +1045,93 @@ resource "snowflake_schema" "test" {
 `
 	return fmt.Sprintf(s, name, database, pipeExecutionPaused)
 }
+
+func TestAcc_Schema_migrateFromV0941_ensureSmoothUpgradeWithNewResourceId(t *testing.T) {
+	id := acc.TestClient().Ids.RandomDatabaseObjectIdentifier()
+
+	resource.Test(t, resource.TestCase{
+		PreCheck: func() { acc.TestAccPreCheck(t) },
+		TerraformVersionChecks: []tfversion.TerraformVersionCheck{
+			tfversion.RequireAbove(tfversion.Version1_5_0),
+		},
+		CheckDestroy: acc.CheckDestroy(t, resources.Schema),
+		Steps: []resource.TestStep{
+			{
+				ExternalProviders: map[string]resource.ExternalProvider{
+					"snowflake": {
+						VersionConstraint: "=0.94.1",
+						Source:            "Snowflake-Labs/snowflake",
+					},
+				},
+				Config: schemaBasicConfig(id.DatabaseName(), id.Name()),
+				Check: resource.ComposeAggregateTestCheckFunc(
+					resource.TestCheckResourceAttr("snowflake_schema.test", "id", helpers.EncodeSnowflakeID(id)),
+				),
+			},
+			{
+				ProtoV6ProviderFactories: acc.TestAccProtoV6ProviderFactories,
+				Config:                   schemaBasicConfig(id.DatabaseName(), id.Name()),
+				Check: resource.ComposeAggregateTestCheckFunc(
+					resource.TestCheckResourceAttr("snowflake_schema.test", "id", id.FullyQualifiedName()),
+				),
+			},
+		},
+	})
+}
+
+func schemaBasicConfig(databaseName string, name string) string {
```

Comment:
We already have `schemav094`.

Reply:
I would leave it, ultimately we want to use the new approach and this name makes more sense. Let's leave both.